app/src/test/java/com/appunite/loudius/ui/pullrequests/PullRequestsViewModelTest.kt

```diff
@@ -40,6 +40,17 @@ class PullRequestsViewModelTest {
     private val pullRequestRepository = spyk(FakePullRequestRepository())
     private fun createViewModel() = PullRequestsViewModel(pullRequestRepository)
 
+    @Test
```

Comment:
suggestion: 
1. we might also have some test for ReviewersViewModel.
2. we might also have a test for `isRefreshing` field.

Reply:
done :)